### PR TITLE
Add accept file input testing

### LINKED ISSUE #20842: Faces 4.0: h:inputFile accept="..."
User Story:
As a user of Open Liberty and the faces-4.0 feature, I want to create forms that restrict the type of file uploaded to a certain extension, format, or explicit name. 

Spec issue: `https://github.com/jakartaee/faces/issues/1556`

See: https://balusc.omnifaces.org/2021/11/whats-new-in-faces-40.html

Test cases:
- [x]  A valid case-insensitive filename extension, starting with a period (".") character. For example: .jpg, .pdf, or .doc.
- [x]  A valid MIME type string, with no extensions.
  - [x]  The string audio/* meaning "any audio file".
  - [x]  The string video/* meaning "any video file".
  - [x]  The string image/* meaning "any image file".
- [x]  All of the above cases, but also with multiple input files.

### PR DESCRIPTION
**Pull request by KyleAure**
Fixes #20842

### CODE REVIEW

**Comment by KyleAure** [comment]
#libby

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#libby check headers. 
This PR is ready for merging once reviewed.

**Comment by KyleAure** [comment]
#build Needs to be rebuilt since it has been so long since the last one.

**Comment by KyleAure** [comment]
#build

File: dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_acceptInputFileTest/server.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing Faces 4.0 and input file section">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>cdi-4.0</feature>
```
Comment (pnicolucci) [inline dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_acceptInputFileTest/server.xml:18]: I merged https://github.com/OpenLiberty/open-liberty/pull/23750 so CDI is enabled automatically now with faces-4.0 no need to specify in the server.xml.

**Comment by pnicolucci** [comment]
@KyleAure  just the one minor review comment. Looks good otherwise. Thanks!

**Review by volosied — APPROVED** [review]
Tested locally. Looks good to me as well. Thanks! :)

**Comment by KyleAure** [comment]
#libby re-check headers

File: dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_acceptInputFileTest/server.xml (hunk 1 above)
Comment (KyleAure) [reply dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_acceptInputFileTest/server.xml:18]: 👍🏼 Thanks for the heads up.  I'll keep this the way it is written, but will remove cdi features from future pulls.

**Pull request merged**